packaging/os/dnf.py

```diff
@@ -258,13 +258,35 @@ def _mark_package_install(module, base, pkg_spec):
         module.fail_json(msg="No package {} available.".format(pkg_spec))
 
 
+def _parse_spec_group_file(names):
+    pkg_specs, grp_specs, filenames = []
```

Comment:
One change needed here:
```
    pkg_specs = []
    grp_specs = [] 
    filenames = []
```

Python raises a ValueError without this change because it's expecting that it's supposed to unpack the right hand side into three variables on the left hand side.

Doing the following would fix the ValueError but it would have a logic error instead:
```
pkg_specs = grp_specs = filenames = []
```
That's because the above is the same as
```
filenames = []
grp_specs = filenames
pkg_specs = grp_specs
```

which sets all three variables to the same list.  Since we want three separate lists, we need to define them all separately.